### DISCUSSION TITLE
Phalcon devtool issue

### DISCUSSION THREAD

**Discussion by rajeshpavasi** [phalcon/cphalcon#16274] [General]
`Hi All,

 I'm getting error while run Phalcon create-project test

Phalcon Dev Tools (4.2.0)

PHP Fatal error:  Uncaught Error: Class "Phalcon\Config" not found in C:\xampp\htdocs\phalcon_dev\vendor\phalcon\devtools\src\Builder\Component\AbstractComponent.php:48

I installed below 

Phalcon Dev Tools (4.2.0)

Environment:
  OS: Windows NT DESKTOP-A40AND7 10.0 build 19044 (Windows 10) AMD64
  PHP Version: 8.0.25
  PHP SAPI: cli
  PHP Bin: C:\xampp\php\php.exe
  PHP Extension Dir: C:\php\ext
  PHP Bin Dir: C:\php
  Loaded PHP config: C:\xampp\php\php.ini
Versions:
  Phalcon Dev Tools Version: 4.2.0
  Phalcon Version: 5.0.0
  Admin LTE Version: 3.0.1

so kindly help me to make success`

**Comment by lmfmaier** [comment]
Devtools have not been updated for Phalcon 5 yet
